starter-code/main.js

```diff
@@ -0,0 +1,145 @@
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result.length);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+              break;
+              case "3":
+              db.collection('companies').find({"founded_year":2004}, {name: 1, _id: 0}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result.length);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+              break;
+              case "4":
+              db.collection('companies').find({"founded_month":2, "founded_year":2004}, {name: 1, _id: 0}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+              break;
+              case "5":
+              db.collection('companies').find({ 
+                  $and:[
+                      {"founded_year":2004},
+                      {"founded_month":{$gte: 4}},
+                      {"founded_month":{$lte: 6}}
+                  ]},
+                      {name: 1, _id: 0}).sort({"founded_month":1, "founded_day":1}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+              break;
+              case "6":
+              db.collection('companies').find({"offices.city":"Barcelona"}, {name: 1, _id: 0}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+              break;
+              case "7":
+              db.collection('companies').find({}, {name: 1, _id: 0}).limit(10).sort({"number_of_employees":-1, "founded_day":1}).toArray((error, result) => {
```

Comment:
👍 
you should have also added the number of employees query:
**db.collection('companies').find({}, {name: 1, number_of_employees: 1}).sort({number_of_employees: -1}).limit(10).toArray((error, result)=>{**